pkg/arvo/app/graph-store.hoon

```diff
@@ -297,6 +297,10 @@
       |^
       =/  [=graph:store mark=(unit mark:store)]
         (~(got by graphs) resource)
+      ::  no-op if we already have any of the nodes
```

Comment:
_Any_? Do we not still want to take action, if some of these are new to us?

Reply:
Correct, the transaction as a whole will have failed in that case.

Reply:
I think I'm missing context, because I don't understand why you don't still want to process the unknown/new content, but I'll believe it.